src/reconcile/rec_write.c

```diff
@@ -2245,8 +2167,10 @@ __rec_write_wrapup(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
     /*
      * If using the lookaside table eviction path and we found updates that weren't globally visible
      * when reconciling this page, copy them into the database's lookaside store.
+     *
+     * If no updates were saved, no need to write to lookaside.
```

Comment:
Can you explain what is the reason or advantage do we get if we don't do it.

Reply:
This is just to reduce the wasted effort to go through all the code in las insert.